test-suite/Neovim/API/THSpec.hs

```diff
@@ -18,8 +18,6 @@ import qualified Data.Map                   as Map
 import           Test.Hspec
 import           Test.QuickCheck
 
-import           Control.Applicative
-
```

Comment:
Removing this might break the GHC versions before the AMP, not too sure if it would work otherwise, but I'd rather keep it here for now.

Reply:
whoops, that wasn't intended, just deleted what hie told me out of habit